From 0cbb39f5b5ba27bf1b992badfae1b812fd8532b1 Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Wed, 25 Oct 2023 16:52:01 +0200
Subject: [PATCH 386/841] fix(st-clock): make ops_array_mp25 static

Issue found with sparse tool:
drivers/st/clk/stm32mp2_clk.c:1050:28: warning: symbol 'ops_array_mp25'
 was not declared. Should it be static?

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git e4bc8251d474dd0e112f0e800d6ffddd066a4ab6 fix(st-clock): make ops_array_mp25 static]
Change-Id: I9d075ed6320b07bc505f30dfb7cfbf8b03d9ec9b
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/338363
---
 drivers/st/clk/stm32mp2_clk.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/st/clk/stm32mp2_clk.c b/drivers/st/clk/stm32mp2_clk.c
index 8c8f71443..d2c4050cd 100644
--- a/drivers/st/clk/stm32mp2_clk.c
+++ b/drivers/st/clk/stm32mp2_clk.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2018-2022, STMicroelectronics - All Rights Reserved
+ * Copyright (C) 2018-2023, STMicroelectronics - All Rights Reserved
  *
  * SPDX-License-Identifier: GPL-2.0+ OR BSD-3-Clause
  */
@@ -1048,7 +1048,7 @@ enum {
 	MP25_LAST_OPS
 };
 
-const struct stm32_clk_ops *ops_array_mp25[MP25_LAST_OPS] = {
+static const struct stm32_clk_ops *ops_array_mp25[MP25_LAST_OPS] = {
 	[NO_OPS] =  NULL,
 	[FIXED_FACTOR_OPS] = &clk_fixed_factor_ops,
 	[GATE_OPS] = &clk_gate_ops,
-- 
2.47.2

